packages/file_selector/file_selector/lib/file_selector.dart

```diff
@@ -9,7 +9,19 @@ import 'package:file_selector_platform_interface/file_selector_platform_interfac
 export 'package:file_selector_platform_interface/file_selector_platform_interface.dart'
     show XFile, XTypeGroup;
 
-/// Open file dialog for loading files and return a file path
+/// Open file dialog for loading files and return a file path.
+///
+/// [acceptedTypeGroups] is the file type that can be selected in the dialog.
+/// there are differences in behavior depending on the platform as follows.
+/// - Windows: Each group will be an entry in a list of filter options.
+/// - macOS: The union of all types allowed by all of the groups will be allowed.
+///
+/// [initialDirectory] is the full path to the directory that will be displayed when the dialog is opened.
+///
+/// [confirmButtonText] is the text in the confirmation button of the dialog.
+/// when omitted, the default OS label. (for example, "Open")
```

Comment:
Comments should use normal grammar; sentences need to start with capital letters. Also, the parenthetical is written outside of the sentence, but isn't a sentence itself.